src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/backlink/BackLink.scala

```diff
@@ -32,6 +32,8 @@ object BackLink {
 
   def defaultObject: BackLink = BackLink()
 
+  def USE_BROWSER_BACK = "#"
```

Comment:
This is just a hack that would allow services using the `hmrcLayout` to pass a dummy URL that will trigger the JS, like
```
@hmrcLayout(backLinkUrl = Some(BackLink.USE_BROWSER_BACK))
```
because passing `None` means the back link isn't rendered at all.  Probably needs modelling properly, but in a way that won't break existing services.

Reply:
I reckon in practice we might want to add another attribute like `backLink=Some(BackLink)` then I could see rather than using the a special url value, have something like this:

```
object BackLink {
   ...   

   val mimickBrowserBackButtonViaJavaScript = BackLink(
     href = "#",
     attributes = "data-module" -> "hmrc-back-link"
   )

   ...
}
```